### ISSUE TITLE
Single-label multi-classification or multi-label multi-classification?

### ISSUE THREAD

**Issue by JingrongZheng** [RL4M/IRENE#7]
I have read the paper and it seemed to be a single-label multi-classification problem. But the code use BCE and sigmoid instead of crossEntropy and softmax. So does it mean that the patient may have more than one lung disease?(I want to confirm it since I am not a medical expert)

**Comment by JingrongZheng** [comment]
It seemed that it is multi-labels multi-classfications. Sorry
